modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -398,6 +406,8 @@ impl ServerModule for Wallet {
 
         items.push(WalletConsensusItem::Feerate(fee_rate_proposal));
 
+        items.push(WalletConsensusItem::PegInMinimum(Amount::from_sats(10_000)));
```

Comment:
Wanted to do this in a follow up. This is just a placeholder value for now - 10_000 is the highest we can set it to before our test would fail, however, not a sufficient minimum which I assume would be at leats 1_000_000 sats.